tests/spec/fs.appendFile.spec.js

```diff
@@ -110,6 +110,19 @@ describe('fs.appendFile', function() {
     });
   });
 
+  it('should error when the symbolic link does not exist', function(done) {
+    var fs = util.fs();
+    var contents = 'This is a file.';
+     
+    fs.appendFile('/newfile', contents, { encoding: 'utf8' }, function(error) {  
+    expect(error).to.exist;
+    expect(error.code).to.equal('ENOENT');
+    expect(result).not.to.exist;
```

Comment:
`result` is being used in here but it has not been declared.

Reply:
Thank you! I am changing that, 
Must be more careful when I am using other test examples.